app/models/user.rb

```diff
@@ -1,2 +1,10 @@
 class User < ApplicationRecord
+  has_many :posts
+
+  has_many :comment, foreign_key: 'author_id'
+  has_many :like, foreign_key: 'author_id'
```

Comment:
- [ ] According to your `schema.rb` file, is only the `posts` table that has an `author_id` as foreign key and the `comments` and `likes` post have `user_id` as foreign_key. Therefore I suggest that you remove the `foreign_key: 'author_id'` property from the `comments` and `likes` association and have it only for the `posts`

- [ ] Kindly notice that the `has_many` association always take the plural of the model. That mean when the model name is `Comment` the association will be `has_many :comments`

**Example:**
```rb
  has_many :posts, foreign_key: 'author_id'
  has_many :comments
  has_many :likes
```